q2_feature_table/plugin_setup.py

```diff
@@ -652,3 +654,59 @@
                 'splits are defined by values in metadata column.',
     examples={}
 )
+
+plugin.methods.register_function(
+    function=q2_feature_table.tabulate_feature_frequencies,
+    inputs={'table': FeatureTable[Frequency]},
```

Comment:
Let's have all these actions work on `FeatureTable[PresenceAbsence]` as well, as these are going to become more common in some of our workflows.